### ISSUE TITLE
[BUG] installing with pnpm throw exception on runtime

### ISSUE THREAD

**Issue by raz-viber** [Gmousse/dataframe-js#93]
**Describe the bug**
after installing dataframe-js with pnpm I tried to do:
`require('dataframe-js').DataFrame`
and got:

> Error: Cannot find module 'regenerator-runtime/runtime'
>     at Function.Module._resolveFilename (module.js:489:15)
>     at Function.Module._load (module.js:439:25)
>     at Module.require (module.js:517:17)
>     at require (internal/module.js:11:18)
>     at Object.<anonymous> (/Users/raz/viber/test/node_modules/.registry.npmjs.org/dataframe-js/1.4.2/node_modules/dataframe-js/lib/dataframe.js:40:1)
>     at Module._compile (module.js:573:30)
>     at Object.Module._extensions..js (module.js:584:10)
>     at Module.load (module.js:507:32)
>     at tryModuleLoad (module.js:470:12)
>     at Function.Module._load (module.js:462:3)

 

**To Reproduce**
- mkdir test
- pnpm init
- pnpm install dataframe-js --save
- run node
- require('dataframe-js').DataFrame

**Expected behavior**
to work...

**Screenshots**

**Desktop (please complete the following information):**
 - OS: osx
 - NodeJS : 8.3

**Smartphone (please complete the following information):**

**Additional context**
package.json:
```
{
  "name": "test",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "dataframe-js": "^1.4.2"
  }
}

```

**P.S**
even after installing 

> babel-runtime

 as suggested (https://github.com/pnpm/pnpm/issues/1435#issuecomment-432410096) did not helped

the flag shamefully-flatten did worked but it not suite for our entire project

**Comment by Gmousse** [comment]
Hi @raz-viber,

Thank you for your report.
I m not a pnpm user, let me get few documentation about this and I will find a fix.

**Comment by Gmousse** [comment]
Hi @raz-viber,
Could you try to test this branch ? `pnpm i git+https://github.com/Gmousse/dataframe-js.git#release/1.4.3`

It seems to work now. Its related to #88 and this babel environment nightmare.

**Comment by raz-viber** [comment]
works like a charm!

**Comment by Gmousse** [comment]
Nice ! I will publish the fix release tommorow

**Comment by Gmousse** [comment]
It must be fixed on the new 1.4.3.
Thanks for your patience
